frontend/badges/legacyUser/localizations.json

```
@@ -1,14 +1,14 @@
 {
   "es": {
     "name": "Usuario Histórico",
-    "description": "El usuario realizó algún envío dentro del primer año desde el lanzamiento de omegaUp."
+    "description": "El usuario ha realizado algún envío durante tres años seguidos."
   },
   "en": {
     "name": "Legacy User",
-    "description": "User made a submission within the first year since omegaUp launchment."
+    "description": "User has made a shipment for three years in a row."
```

```suggestion
    "description": "User has made a submission for three years in a row."
```

ay, se me fue esto. pero según yo debería ser lo último!

---

listo